pkg/liqonet/utils/utils.go

```diff
@@ -245,3 +245,14 @@ func GetFirstIP(network string) (string, error) {
 	}
 	return firstIP.String(), nil
 }
+
+// MustGetFirstIP returns the first IP address of a network.
+// It panics in case of errors.
+func MustGetFirstIP(network string) string {
```

Comment:
Is this function called only for testing purposes? Since it panics in case of error I would move it on the testing package where it is called.